docs/index.md

```diff
@@ -0,0 +1,81 @@
+---
+hide:
+  - navigation
+---
+
+# Nautobot Golden Configuration
```

Comment:
If this is the same as readme, need to ci ensure it is the same and possibly some script to generate this.

Reply:
It is and I've updated the links so that we can just include the README in the documentation tree.